collector/cluster-insight-controller.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Not sure if I can add comments to the replication controller file - but I will update the readme to specify how to use it.